### PR TITLE
Cleanup of EvictionTest

### PR DESCRIPTION
**Pull request by Donnerbart**
Made use of the fluent config API in the test, which uncovered an unused
configuration. This was hidden since the config was used to retrieve a
`MapConfig`, but not to create the HZ instance. With the fluent API the
usage is easier to see.

### CODE REVIEW

File: hazelcast/src/test/java/com/hazelcast/map/EvictionTest.java [hunk 1]
```diff
@@ -443,20 +423,21 @@ protected void testEvictionLFUInternal(boolean disableStats) {
...
-        cfg.setProperty(GroupProperty.PARTITION_COUNT.getName(), "1");
-        MapConfig mc = cfg.getMapConfig(mapName);
-        mc.setEvictionPolicy(EvictionPolicy.LFU);
-        mc.setEvictionPercentage(90);
-        MaxSizeConfig msc = new MaxSizeConfig();
-        msc.setMaxSizePolicy(MaxSizeConfig.MaxSizePolicy.PER_NODE);
-        msc.setSize(size);
-        mc.setMaxSizeConfig(msc);
-
-        HazelcastInstance node = createHazelcastInstance();
```
Comment (Donnerbart) [inline hazelcast/src/test/java/com/hazelcast/map/EvictionTest.java]: A bit hard to spot after all the cleanups, but here was the `createHazelcastInstance()` call which didn't use the created `Config`.

**Comment by Donnerbart** [comment]
run-lab-run

**Comment by Donnerbart** [comment]
run-lab-run

**Comment by mmedenjak** [comment]
run-lab-run

**Pull request merged**